src/utilities/install.js

```diff
@@ -16,18 +16,26 @@ export async function installDeps(choice, name) {
 	const pkg = await findUp('package.json', {
 		cwd: process.cwd() + '/' + name,
 	});
+
 	if (!pkg) throw new Error('No package.json found!');
+
 	const output = JSON.parse(await readFile(pkg, 'utf8'));
+
```

Comment:
remove space

Reply:
keep if statement and variable together
ie : 
const deps = output.dependencies;
if(!deps) throw Error('fdsf');